src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ActionOfImportItinProposalAction": {
+      "type": "object",
+      "properties": {
+        "emtAction": {
+          "$ref": "#/definitions/ImportItinProposalAction"
+        }
+      }
+    },
+    "AgencyProposal": {
+      "type": "object",
+      "required": [
+        "agencyProposalEntries",
+        "booked",
+        "importDate"
+      ],
+      "properties": {
+        "agencyProposalEntries": {
+          "type": "array",
+          "description": "List of the proposal entries",
+          "items": {
+            "$ref": "#/definitions/AgencyProposalEntry"
+          }
+        },
+        "agencyProposalType": {
+          "type": "string",
+          "description": "The agency proposal type - equal API for generic proposition",
+          "enum": [
+            "CWT",
+            "CWTF",
+            "AEBT",
+            "API",
+            "NONE"
+          ]
+        },
+        "approvalLimitDate": {
+          "type": "string",
+          "description": "The date by which the Request must be approved. This element appears only when integrated with Concur Travel"
+        },
+        "autoSelect": {
+          "type": "boolean",
+          "example": false,
+          "description": "The option AutoSelect permit to choose automaticly a proposal"
+        },
+        "booked": {
+          "type": "boolean",
+          "example": false,
+          "description": "True if this travel is (or has to be) handled by a travel agency"
+        },
+        "comments": {
+          "type": "string",
+          "description": "Comment",
+          "minLength": 0,
+          "maxLength": 2000
+        },
+        "importDate": {
+          "type": "string",
+          "description": "The date of the import"
+        },
+        "itineraryLocator": {
+          "type": "string",
+          "description": "Itinerary Locator",
+          "minLength": 0,
+          "maxLength": 32
+        },
+        "policyCompliant": {
+          "type": "boolean",
+          "example": false,
+          "description": "The policy Compliant"
+        },
+        "proposal": {
+          "type": "boolean",
+          "example": false,
+          "description": "True if the status is proposal and false if the status is confirmed or Ticket issued"
+        },
+        "proposalBatchSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The size of the batch of proposal expected to be received.",
+          "minimum": 1,
+          "exclusiveMinimum": false
+        },
+        "proposalOrder": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The position of the proposal",
+          "minimum": 1,
+          "exclusiveMinimum": false
+        },
+        "proposalUuid": {
+          "type": "string",
+          "description": "Unique Identifier of Agency Proposal receive"
```

Comment:
Unique Identifier of Agency Proposal receive > Unique Identifier of Agency Proposal

Reply:
Idem as above, agency proposals not yet externalized in Request v4 APIs. Please remove if not part of https://developer.concur.com/api-reference/request/v4.endpoints.html